GOVERNANCE.md

```diff
@@ -0,0 +1,136 @@
+* Note: having your name in CODEOWNERS does not give any special permission, but it does mean that you will be notified whenever there is a new PR with changes to that file. 
+
+## Committer
+
+* Definition: permission to commit to, and merge PRs into, master branch. 
+* How to obtain this role: after a reviewer has a consistent history of careful reviews of others' PRs, then a current Committer should ask all other current Committers if they approve promoting the Reviewer to Committer, and it should be done if there is Consensus among active Committers. 
+* How this role is recognized: credited via role="aut" in DESCRIPTION (so they appear in Author list on CRAN), and added to https://github.com/orgs/Rdatatable/teams/maintainers which gives permission to merge PRs into master branch. 
+
+## CRAN maintainer
+
+* Definition: in charge of communication with CRAN. Responsible for submitting releases to CRAN on a regular basis, and for responding to requests from CRAN.
+* How to obtain this role: (1) merge into master a PR adding role="cre" to DESCRIPTION, and (2) submit updated package to CRAN (previous CRAN maintainer will have to confirm change by email to CRAN). 
+* How this role is recognized: credited via role="cre" in DESCRIPTION, so they appear as Maintainer on CRAN. 
+
+## Other roles
+
+Each of the below roles is important, and a wiki page should be
+created to explain (1) details of responsibilities, (2) who to contact
+to obtain this role. 
+
+* Triage manager (to decide which issues/PRs to include in next releases), no special permissions.
+* Translation manager (to communicate with translators, and source code changes related to i18n, for example making sure messages are translation-ready, updating the .pot template), no special permissions (updates should happen in a pull request as usual).
+* Performance testing manager (to prevent performance regressions), no special permissions (updates should happen in a pull request as usual).
+* Continuous integration/deployment manager (to maintain script which run tests and build binaries of development branches for user testing before release), no special permissions (updates should happen in a pull request as usual).
+* Reverse dependency manager (to ensure compatibility with other CRAN packages), need access to a computer that can run revdep checks in parallel.
+
+# Decision-making processes
+
+## Definition of Consensus
+
+Most decisions in the project happen by Consensus, which means that no active people (typically Reviewers and/or Committers) have expressed major blocking concerns, in a public discussion (typically in a GitHub issue or pull request). In Consensus, non-response by inactive members indicates tacit agreement.
+
+## Pull Requests
+
+A pull request can be merged as long as there is one approving review (ideally from a reviewer of the affected files, different person from author of PR), and Consensus from active Reviewers and Committers.
```

Comment:
Since consensus is also reached by public silence we might want to raise the barrier from 1 reviewer to 1 reviewer + 1 committer and lift the different person constraint?

Or is the committer always supposed to review except if own PR?

Reply:
hi does this clarify your concern?
```markdown
A pull request can be merged by any committer, if there is one approving review, and Consensus from active Reviewers and Committers. 
* approving review must come from different person from author of PR.
* approving review ideally comes from a reviewer of the affected files.
* approving review can be same as committer who merges the PR.
```

Reply:
Sounds good, ty